src/components/SearchBar.tsx

```diff
@@ -41,11 +79,32 @@ export default function SearchBar() {
         id={id}
         value={value}
         type="search"
-        placeholder="Enter terms"
+        placeholder="Enter file or folder name"
         onChange={(event) => {
           setValue(event.target.value)
         }}
       />
+      {
+        searchText.length > 0 ?
+        <StyledP>{searchResults.length} results</StyledP>
+        : null
+      }
+      {searchResults.map(result => {
+        return (
+          <React.Fragment key={result.path}>
+            <SearchResultButton title={result.path} value={result.path} onClick={() => onClick(result)}>
+              <SearchResultSpan>
+                {result.type === "tree" ? 
+                  <FontAwesomeIcon icon={faFolderOpen} />
```

Comment:
Could we maybe style the icons to be a little more greyish? The pure black is a bit harsh.